tests/test_ahd_to_list.py

```diff
@@ -30,9 +32,10 @@ def get_empty_document_reference():
     return doc_ref
 
 
+@mock.patch.dict(os.environ, {"ahd_version": "5.0"})
```

Comment:
might be cleaner to use FastAPI's dependency injection to pass a "Settings" object to a mapper class https://github.com/miracum/ahd2fhir/pull/41 and override this object's parameters instead.